tests/mypy_function_wrapper_t6.py:19: error: Argument 2 to "FunctionWrapper" has incompatible type "str"; expected "Callable[[Callable[[VarArg(Any), KwArg(Any)], Any], Any, tuple[Any, ...], dict[str, Any]], Any]"  [arg-type]
tests/mypy_function_wrapper_t6.py:20: error: Argument 2 to "FunctionWrapper" has incompatible type "None"; expected "Callable[[Callable[[VarArg(Any), KwArg(Any)], Any], Any, tuple[Any, ...], dict[str, Any]], Any]"  [arg-type]
Found 2 errors in 1 file (checked 1 source file)
